services/brig/src/Brig/API/Public.hs

```diff
@@ -617,12 +619,19 @@ getRichInfo self user = do
 getClientPrekeys :: UserId -> ClientId -> (Handler r) [Public.PrekeyId]
 getClientPrekeys usr clt = lift (wrapClient $ API.lookupPrekeyIds usr clt)
 
-newNonce :: UserId -> (Handler r) Nonce
+newNonce :: UserId -> (Handler r) (Nonce, CacheControl)
```

Comment:
why?  why not hard-wire the cache-control into the end-point?

Reply:
I like this because this is more explicit.